viewer-admin/src/main/java/nl/b3p/viewer/admin/stripes/LayoutManagerActionBean.java

```diff
@@ -213,12 +213,6 @@ public Resolution view() throws JSONException {
     public Resolution config() {
         EntityManager em = Stripersist.getEntityManager();
 
-        try {
-            appConfigJSON = application.toJSON(context.getRequest(),false,
-                    true, false, false,em, false).toString();
```

Comment:
Why was it here? seems this is the only place it's used in this class and I can't find any other use... 
I think you could remove `appConfigJSON` from the class altogether

Reply:
Done